README_64bit.md

```diff
@@ -52,7 +52,7 @@ How to use Auto-Config with the 64-bit ROMs
 
 <sup>2</sup> Recommended USB ports for flashing (use other if first doesn't work):
  - On TX9S, flash with the USB port closest to the network port
- - On X96 Mini use left USB port (recommended on Discord)
+ - On X96 Mini use port farther away from SD-card slot.
  - On A95X F1 use right USB port (recommended on Discord)
```

Comment:
And here

Reply:
If you have A95X F1 that you flashed lately and know which ports are which just submit the PR. I don't have access to that and changed only the x96 mini that I have in front of me atm.

Reply:
A95x for burning closest to power connection for autoconfig closest to SD

Reply:
Thanks @dkmur, updated.